bcda/api.go

```diff
@@ -53,6 +54,9 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+//Setting a default here.  It may need to change.  It also shouldn't really be used much.
+const BCDA_FHIR_MAX_RECORDS = 10000
```

Comment:
Highly recommend naming this something different.  It is a bit confusing that it is named exactly the same as the env var.

Reply:
I named it that way on purpose so it would be clear that it represented the same thing.  I can change it to BCDA_FHIR_MAX_RECORDS_DEFAULT